src/main/java/it/gov/pagopa/payhub/activities/service/exportflow/debtposition/PaidExportFlowFileService.java

```diff
@@ -0,0 +1,105 @@
+                                     @Value("${export-flow-files.paid.filename-prefix}")String filenamePrefix,
+                                     @Value("${export-flow-files.paid.page-size}") int pageSize,
+                                     ExportFileService exportFileService,
+                                     DataExportService dataExportService,
+                                     InstallmentExportFlowFileDTOMapper installmentExportFlowFileDTOMapper
+                                     ) {
+
+        super(csvService, csvRowDtoClass, fileArchiverService, workingDirectory, relativeFileFolder, filenamePrefix);
+        this.pageSize = pageSize;
+        this.exportFileService = exportFileService;
+        this.dataExportService = dataExportService;
+        this.installmentExportFlowFileDTOMapper = installmentExportFlowFileDTOMapper;
+    }
+
+    @Override
+    public PaidExportFile findExportFileRecord(Long exportFileId) {
+        return exportFileService.findPaidExportFileById(exportFileId)
+                .orElseThrow(() -> new ExportFlowFileNotFoundException("Cannot found paidExportFile having id: %d".formatted(exportFileId)));
+    }
+
+    @Override
+    public Long getOrganizationId(PaidExportFile exportFile) {
+        return exportFile.getOrganizationId();
+    }
+
+    @Override
+    protected String getFlowFileVersion(PaidExportFile exportFile) {
+        return exportFile.getFlowFileVersion();
+    }
+
+    @Override
+    public List<InstallmentPaidViewDTO> retrievePage(PaidExportFile exportFile, PaidExportFileFilter filter, int pageNumber) {
+        List<InstallmentPaidViewDTO> installmentPaidViewDTOList = Collections.emptyList();
+
+        PagedInstallmentsPaidView pagedInstallmentsPaidView = dataExportService.exportPaidInstallments(exportFile.getOrganizationId(), exportFile.getOperatorExternalId(), filter, pageNumber, pageSize, List.of("installmentId"));
+
+        if (pagedInstallmentsPaidView != null){
+            installmentPaidViewDTOList= pagedInstallmentsPaidView.getContent();
+        }
+
+        return installmentPaidViewDTOList;
+    }
+
+    @Override
+    protected ExportFileStatus getExportStatus(PaidExportFile exportFile) {
+        return exportFile.getStatus();
+    }
+
+    @Override
+    public PaidInstallmentExportFlowFileDTO map2Csv(InstallmentPaidViewDTO retrievedInstallment) {
+        return installmentExportFlowFileDTOMapper.map(retrievedInstallment);
+    }
+
+    @Override
+    protected PaidExportFileFilter getExportFilter(PaidExportFile exportFile) {
+        PaidExportFileFilter filterFields = exportFile.getFilterFields();
+
+        return PaidExportFileFilter.builder()
+                .paymentDate(filterFields != null ? filterFields.getPaymentDate() : null)
+                .debtPositionTypeOrgId(filterFields != null ? filterFields.getDebtPositionTypeOrgId() : null)
+                .build();
```

Comment:
why this and not just
```suggestion
        return exportFile.getFilterFields();
```